### ISSUE TITLE
error occurred on windows 

### ISSUE THREAD

**Issue by aimen08** [NvChad/NvChad#535]
**Describe the bug**
i did clean install of windows , tried to install nvim with nvChad , an error occurred  (cannot resolve symbol 'build_stl_str_hl').

**To Reproduce**
1.install nvim and nvchad ,using windows terminal Version: 1.10.2714.0

**Screenshots**
![image](https://user-images.githubusercontent.com/32209952/136120394-e98a9df1-cc55-49c1-be2b-ec5976466371.png)

**Desktop (please complete the following information):**
 - windows 10 pro ver 21H1 (19043)
 - Terminal Windows terminal (1.10.2714.0)
 - Version of Neovim (v0.5.1)

**Comment by siduck76** [comment]
did you update nvchad to the latest + did packersync?

**Comment by aimen08** [comment]
yeah i did , i cloned it from here , i also tried to look up the error on feline and found nothing related.

**Comment by barungh** [comment]
I just came here to report the same issue
